tests/testthat/test_kernel.r

```diff
@@ -0,0 +1,6 @@
+context('kernel')
+
+status <- system2('python3', 'test_ir.py', wait=TRUE)
```

Comment:
we add `python3` and `jupyter_kernel_test` to `SystemRequirements` with an explanation that they’re needed for tests.